packages/amazonq/src/lsp/client.ts

```diff
@@ -25,24 +35,27 @@ export async function startLanguageServer(
 
     const serverModule = resourcePaths.lsp
 
+    const argv = [
+        '--nolazy',
+        '--preserve-symlinks',
+        '--stdio',
+        '--pre-init-encryption',
+        '--set-credentials-encryption-key',
+    ]
     const serverOptions = createServerOptions({
```

Comment:
I forgot that we have 2 of these modules. Will this module (`packages/amazonq/src/lsp/client.ts`) replace `packages/core/src/amazonq/lsp/lspClient.ts` @jpinkney-aws  ?

Reply:
`lsp/client.ts` is the amazon q language server, `lsp/lspClient.ts` is the workspace context language server

Reply:
We may end up deprecating the `lsp/lspClient.ts` one eventually

Reply:
> We may end up deprecating the `lsp/lspClient.ts` one eventually now that workspace context is technically in flare

yes.  also, this logic is pretty similar yet lives in 2 different places